traffic_ops/app/db/migrations/20200727000000_add_deliveryservice_service_category.sql

```diff
@@ -0,0 +1,26 @@
+/*
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+	    http://www.apache.org/licenses/LICENSE-2.0
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+*/
+
+-- +goose Up
+-- SQL in section 'Up' is executed when this migration is applied
+CREATE TABLE IF NOT EXISTS service_category (
+                                                id BIGSERIAL PRIMARY KEY,
+                                                name TEXT UNIQUE NOT NULL CHECK (name <> ''),
```

Comment:
If name is unique and not null, do these really need an id? I know this has been out a while, and I won't block the PR for this, but it seems like the whole concept could be a lot simpler by having only one unique key.

Reply:
I was unable to find bandwidth for this change. If you feel strongly about it, I can get it prioritized but it may take a little longer.